test/js/bun/net/socket.test.ts

```diff
@@ -1,13 +1,13 @@
 // @known-failing-on-windows: 1 failing
 import { expect, it } from "bun:test";
-import { bunEnv, bunExe, expectMaxObjectTypeCount } from "harness";
+import { bunEnv, bunExe, expectMaxObjectTypeCount, isWindows } from "harness";
 import { connect, fileURLToPath, SocketHandler, spawn } from "bun";
 
 it("should coerce '0' to 0", async () => {
   const listener = Bun.listen({
     // @ts-expect-error
     port: "0",
-    hostname: "localhost",
```

Comment:
We should not need to do this 

Please fix the underlying reason for this being necessary instead of applying this bandaid.

Reply:
is not strict necessary but localhost can be ipv6 or ipv4 so I just choose a ipv4 instead of localhost.

Reply:
The bug here is that it didn’t pick the right one. This change shouldn’t be necessary

Reply:
agreed